sfputil/main.py

```diff
@@ -894,26 +933,29 @@ def set_lpmode(logical_port, enable):
         click.echo("Error: No physical ports found for logical port '{}'".format(logical_port))
         return
 
-    if len(physical_port_list) > 1:
-        ganged = True
+    if is_rj45_port_from_api(logical_port):
```

Comment:
Should we keep the same type in function "def fwversion" that we check the port type and just exit if it is RJ45? In that case, we don't need to change the indent here and the diff would be clearer. No strong opinion.